pkg/remotewrite/prometheus.go

```diff
@@ -141,3 +149,56 @@ func MapTrend(series metrics.TimeSeries, t time.Time, sink *trendSink) []*prompb
 		},
 	}
 }
+
+// MapTrendAsNativeHistogram maps the Trend type to the experimental Native Histogram.
+func MapTrendAsNativeHistogram(series metrics.TimeSeries, t time.Time, sink *trendSink) []*prompb.TimeSeries {
+	labels := MapTagSet(series.Tags)
+	timestamp := t.UnixMilli()
+
+	return []*prompb.TimeSeries{
+		{
+			// TODO: should we map the builtin metrics
+			// with the Prometheus' convention?
+			//
+			// e.g. k6_http_reqs_duration_seconds
```

Comment:
I wish I'd say that we should follow the Prometheus convention. Still, on the other side, I think we have a lot of the documentation that remains referring to it as the `k6_http_reqs_duration`, so if we adjust it, we should somehow explain it in other places, so not sure what we should do, but maybe raise it in a wider-team discussion.

Reply:
>so if we adjust it, we should somehow explain it in other places

Just for clarification, this would be only for the Prometheus Output, so it would be visible only from a Prometheus server (or any other compatible storage). In k6 it would continue to remain the same current name.

So, do you have some examples of other places where you would like to mention it?

Reply:
That's what I got, but still, if we change the name, it should be communicated. I know that we also attach prefixes, but generally, the name change looks too big for me :shrug: 

Maybe I'm over-thinking there :sweat_smile:

Reply:
Ofc, we can communicate them in the release notes of the new version. I don't see other places where we could announce them. Also, this is an extension so there isn't a stability guarantee.

Reply:
Okay, then let's consider that I don't have a strong opinion here apart from the explicitly communicating and documenting what ever will be result :+1: